lnwallet/chainfee/estimator.go

```diff
@@ -324,43 +332,35 @@ func NewBitcoindEstimator(rpcConfig rpcclient.ConnConfig, feeMode string,
 // NOTE: This method is part of the Estimator interface.
 func (b *BitcoindEstimator) Start() error {
 	// Once the connection to the backend node has been established, we'll
-	// query it for its minimum relay fee. Since the `getinfo` RPC has been
-	// deprecated for `bitcoind`, we'll need to send a `getnetworkinfo`
-	// command as a raw request.
+	// query it for its minimum relay fee. This is done here so that the
+	// default minFeePerKW can be set.
+	return b.minFeePerKW.initMinRelayFee()
+}
+
+func (b *BitcoindEstimator) fetchMinRelayFee() (SatPerKWeight, error) {
+	// Since the `getinfo` RPC has been deprecated for `bitcoind`, we'll
+	// need to send a `getnetworkinfo` command as a raw request.
 	resp, err := b.bitcoindConn.RawRequest("getnetworkinfo", nil)
 	if err != nil {
-		return err
+		return 0, err
 	}
 
 	// Parse the response to retrieve the relay fee in sat/KB.
 	info := struct {
 		RelayFee float64 `json:"relayfee"`
 	}{}
 	if err := json.Unmarshal(resp, &info); err != nil {
-		return err
+		return 0, err
 	}
 
 	relayFee, err := btcutil.NewAmount(info.RelayFee)
 	if err != nil {
-		return err
+		return 0, err
```

Comment:
So I think we actually want to read `MempoolMinFee` here: https://chainquery.com/bitcoin-cli/getmempoolinfo